modules/core/src/v2/internal/tssUtils.ts

```diff
@@ -285,4 +340,153 @@ export class TssUtils {
 
     return keychains;
   }
+
+  async signTSSTxRequest(
```

Comment:
add documentation:
```
Signs and sends the input Transaction request.
```

Reply:
Working on it, all js doc is missing yet